gpu/containers/include/pcl/gpu/containers/impl/device_array.hpp

```diff
@@ -34,92 +34,258 @@
  *  Author: Anatoly Baskeheev, Itseez Ltd, (myname.mysurname@mycompany.com)
  */
 
-
 #ifndef PCL_GPU_CONTAINER_DEVICE_ARRAY_IMPL_HPP_
 #define PCL_GPU_CONTAINER_DEVICE_ARRAY_IMPL_HPP_
 
+namespace pcl {
 
-namespace pcl
-{
+namespace gpu {
 
-namespace gpu
-{
+/////////////////////  Inline implementations of DeviceArray
+///////////////////////////////////////////////
```

Comment:
Delete some slash to keep this on one line? More instances in this changeset

Reply:
Oh yes, that is a good idea! I tried to address that.